public/styles.css

```diff
@@ -13,19 +13,21 @@ nav {
     font-size: 20px;
     font-weight: 700;
     position: relative;
+    z-index: -100;
+    background: #333;
 }
 
-nav::after {
+nav::before {
     content: '';
-    z-index: -1;
-    background-image: url('images/logo-brick-small.png');
+    background-image: url('images/header/logo-brick-small.png');
```

Comment:
I forgot to update these paths, whoops. Moved all the logo stuff into the header folder.